exercises/practice/forth/forth.odin

```diff
@@ -0,0 +1,45 @@
+package forth
+
+import "core:slice"
+import "core:strconv"
+import "core:strings"
+
+// Forth is a low-level stack-based programming language. Words (primitives and user-defined)
+// are kept in a dictionary and pretty-much every word can be redefined.
+//
+// See https://skilldrick.github.io/easyforth/ (chapters: Introduction, Adding Some Numbers,
+// Defining Words, and Stack Manipulations for material in this exercise).
```

Comment:
Is this information missing from the introduction? If yes, it could be put in `introduction.append.md`